ts/src/base/functions/number.ts

```diff
@@ -149,7 +149,7 @@ const decimalToPrecision = (
                 x = x - missing;
             }
         }
-        return decimalToPrecision (x, ROUND, newNumPrecisionDigits, DECIMAL_PLACES, paddingMode);
+        return decimalToPrecision (x, roundingMode, newNumPrecisionDigits, DECIMAL_PLACES, paddingMode);
```

Comment:
Does it fix any of your tests? If not, I'm not sure we should change this. Also, why didn't you change any of occurrences above? L131, L132, L127, etc

Reply:
It fixes the tests. Lines 127 to 132 are different since it's the floating point error calculation. You can see that if `roundingMode == TRUNCATE` it should be handled in line 148. As I said later I'm quite certain that there was a bug here, in the case where there is no floating point error, calling  `decimalToPrecision(x, TRUNCATE, 0.001, TICK_MODE, 1)` is the same as  `decimalToPrecision(x, ROUND, 4, DECIMAL_PLACES, 1)` which is not the expected behavior and result.